src/components/Button/Button.tsx

```diff
@@ -0,0 +1,125 @@
+import { ReactElement, ReactNode } from 'react'
+import { Button as AntButton } from 'antd'
+
+import styles from './Button.module.css'
+
+const { ghost } = styles
+
+export type ButtonProps = {
+
+  /**
+   * The children nodes to be rendered with the button context
+   */
```

Comment:
can this comment be done in a single line to reduce the file length?

Reply:
We could, however conventionally I think this is the default way of documenting props and methods.
@Giogia what do you think about it?

Reply:
If we follow the [tsdoc](https://tsdoc.org/) comment format (/** * */) we gain some useful documentation features. There is a list of decorating tags that can be used to provide examples, verbal explanations, or links. 

Tsdoc also provides an oneliner format (/** comment */) we can use. 

However, I would rather have a common format for all the component props comments. Since it is pretty common to have multiple lines for a component property comment, I think it is better if we keep the multiline format for every property.